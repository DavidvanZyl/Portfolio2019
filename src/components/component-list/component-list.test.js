import React from "react"
import renderer from "react-test-renderer"

import List from "./component-list"

describe("List", () => {
  test("renders correctly", () => {
    const wrapper = shallow(<List items={[]} children={`<div></div>`} />)
    expect(wrapper).toMatchSnapshot()
  })
})
