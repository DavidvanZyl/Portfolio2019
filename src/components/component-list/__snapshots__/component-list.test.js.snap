// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List renders correctly 1`] = `
<div
  className="list"
>
  <h1 />
  <div
    className="list__item"
  />
</div>
`;
